platforms/core-runtime/build-state/src/main/java/org/gradle/internal/session/BuildSessionScopeServices.java

```diff
@@ -110,118 +62,9 @@ void configure(ServiceRegistration registration, List<PluginServiceRegistry> plu
         registration.add(BuildRequestMetaData.class, buildRequestMetaData);
         registration.add(BuildClientMetaData.class, buildClientMetaData);
         registration.add(BuildEventConsumer.class, buildEventConsumer);
-        registration.add(CalculatedValueContainerFactory.class);
```

Comment:
Moved some service registration back to core, where the services are implemented.